projektrouska/views.py

```diff
@@ -1,568 +1,183 @@
-import json
-from django.http import HttpResponse
-from django.views.decorators.http import require_GET
-from django.http import JsonResponse
+# import json
+import requests
+
 from datetime import datetime, timedelta
 
+from django.http import HttpResponse
+
+# from django.views.decorators.http import require_GET
+# from django.http import JsonResponse
 from django.shortcuts import render
 from django.db import connection
+
 from projektrouska.aktualnost import kontrola
 from projektrouska.settings import DEV
-from projektrouska.functions import *
-from projektrouska.api import *
-
-dni_dopredu = 7  # urcuje v horizontu kolika dni se maji zobrazovat nadchazející opatreni
-
-def indikator_aktualnost():
```

Comment:
Tohle není nikde používané. Je to ještě potřeba? Nebo se to bude používat v budoucnu? Nevím, jestli jsem to měl smazat..